[Live Range Selection] editing/caret/emoji.html and editing/deleting/delete-emoji*.html fail

rniwa opened the pull request:
#### 9ac23cd6dc251efa6fac9a8c1a46c1fb9df55bcc
<pre>
[Live Range Selection] editing/caret/emoji.html and editing/deleting/delete-emoji*.html fail
<a href="https://bugs.webkit.org/show_bug.cgi?id=246265">https://bugs.webkit.org/show_bug.cgi?id=246265</a>

Reviewed by Darin Adler.

Remove the superfluous new line after emoji&apos;s so that canonicalization wouldn&apos;t impact the test result.
This is necessary to achieve the same result before/after enabling live range selection.

* LayoutTests/editing/caret/emoji.html:
* LayoutTests/editing/deleting/delete-emoji-1.html:
* LayoutTests/editing/deleting/delete-emoji-2.html:
* LayoutTests/editing/deleting/delete-emoji-3.html:
* LayoutTests/editing/deleting/delete-emoji-4.html:
* LayoutTests/editing/deleting/delete-emoji-5.html:
* LayoutTests/editing/deleting/delete-emoji-6.html:
* LayoutTests/editing/deleting/delete-emoji-7.html:
* LayoutTests/editing/deleting/delete-emoji-8.html:
* LayoutTests/editing/deleting/delete-emoji-9.html:
* LayoutTests/editing/deleting/delete-emoji.html:
* LayoutTests/platform/gtk/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/gtk/editing/deleting/delete-emoji-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-2-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-3-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-4-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-5-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-6-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-7-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-8-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-9-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-expected.txt:
* LayoutTests/platform/mac/editing/caret/emoji-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-2-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-3-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-4-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-5-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-6-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-7-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-8-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-9-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-expected.txt:
* LayoutTests/platform/wincairo/editing/caret/emoji-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-2-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-3-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-4-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-5-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-6-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-7-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-8-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-9-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-expected.txt:

Canonical link: <a href="https://commits.webkit.org/255353@main">https://commits.webkit.org/255353@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f587a9d85061f032142ba894dbbaca0438e0785c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/92181 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/1410 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/22760 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/101969 "Built successfully") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/162217 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/1408 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/29806 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/84623 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/98135 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/97837 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/915 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/78704 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/27858 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/82834 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/82483 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/70898 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/36228 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/16453 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/33986 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/17610 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/37855 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/40251 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1672 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/39755 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/36743 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/f587a9d85061f032142ba894dbbaca0438e0785c. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/5176

rniwa committed:
[Live Range Selection] editing/caret/emoji.html and editing/deleting/delete-emoji*.html fail
https://bugs.webkit.org/show_bug.cgi?id=246265

Reviewed by Darin Adler.

Remove the superfluous new line after emoji's so that canonicalization wouldn't impact the test result.
This is necessary to achieve the same result before/after enabling live range selection.

* LayoutTests/editing/caret/emoji.html:
* LayoutTests/editing/deleting/delete-emoji-1.html:
* LayoutTests/editing/deleting/delete-emoji-2.html:
* LayoutTests/editing/deleting/delete-emoji-3.html:
* LayoutTests/editing/deleting/delete-emoji-4.html:
* LayoutTests/editing/deleting/delete-emoji-5.html:
* LayoutTests/editing/deleting/delete-emoji-6.html:
* LayoutTests/editing/deleting/delete-emoji-7.html:
* LayoutTests/editing/deleting/delete-emoji-8.html:
* LayoutTests/editing/deleting/delete-emoji-9.html:
* LayoutTests/editing/deleting/delete-emoji.html:
* LayoutTests/platform/gtk/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/gtk/editing/deleting/delete-emoji-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-2-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-3-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-4-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-5-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-6-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-7-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-8-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-9-expected.txt:
* LayoutTests/platform/ios/editing/deleting/delete-emoji-expected.txt:
* LayoutTests/platform/mac/editing/caret/emoji-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-2-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-3-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-4-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-5-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-6-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-7-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-8-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-9-expected.txt:
* LayoutTests/platform/mac/editing/deleting/delete-emoji-expected.txt:
* LayoutTests/platform/wincairo/editing/caret/emoji-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-1-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-2-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-3-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-4-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-5-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-6-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-7-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-8-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-9-expected.txt:
* LayoutTests/platform/wincairo/editing/deleting/delete-emoji-expected.txt:

Canonical link: https://commits.webkit.org/255353@main

webkit-commit-queue commented:
Committed 255353@main (9ac23cd6dc25): <https://commits.webkit.org/255353@main>

Reviewed commits have been landed. Closing PR #5176 and removing active labels.
